lib/node_modules/@stdlib/blas/ext/base/gnannsumkbn/lib/ndarray.js

```diff
@@ -68,6 +72,11 @@ function gnannsumkbn( N, x, strideX, offsetX, out, strideOut, offsetOut ) {
 		out[ offsetOut+strideOut ] = 0;
 		return out;
 	}
+	ox = arraylike2object( x );
+	oo = arraylike2object( out );
+	if ( ox.accessorProtocol || oo.accessorProtocol ) {
+		return accessors( N, ox, strideX, offsetX, oo, strideOut, offsetOut );
```

Comment:
```suggestion
		accessors( N, ox, strideX, offsetX, oo, strideOut, offsetOut );
		return out;
```

Return original output array, not the object.